app/assets/stylesheets/users.scss

```diff
@@ -151,6 +151,7 @@
   border-radius: 50%;
   height: 150px;
   width: 150px;
+  object-fit: cover;
```

Comment:
You might want to test this on different screen sizes to verify it doesn’t crop critical parts of user images unexpectedly.

Reply:
How about setting up a simple crop to make sure user themselves select the highlight of image to get that in-frame?

Reply:
Sounds good

Reply:
Alright, I'll scrap this PR and remake a new PR including all the pfp and user name fixes all across the website.

Reply:
I think it's better to iterate over this pr right ? So that if any maintainers wanna revert any changes that you made on my suggestion, you can rebase from that commit

Reply:
Yeah but I think adding a different option in user profile section is outside the scope of this PR, isn't?